src/lib.rs

```diff
@@ -73,13 +73,15 @@ pub struct MemoryStats {
     /// metric on each platform:
     /// - **Linux, Android, MacOS, iOS**: Resident Set Size
     /// - **Windows**: Working Set
+    /// Data is in bytes
```

Comment:
Honestly this could be folded into the first sentence, ie `The "physical" memory used by this process, in bytes.` and `The "virtual" memory used by this process, in bytes.`